services/brig/src/Brig/API/User.hs

```diff
@@ -524,15 +529,23 @@ sendActivationCode emailOrPhone loc call = case emailOrPhone of
             Just (Just uid, c) -> sendActivationEmail   ek c        uid -- User re-requesting activation
 
     Right phone -> do
-        pk <- maybe (throwE $ InvalidRecipient (userPhoneKey phone))
-                    (return . userPhoneKey)
+        -- validatePhone returns the canonical E.164 phone number format
+        canonical <- maybe (throwE $ InvalidRecipient (userPhoneKey phone))
+                    (return)
```

Comment:
- brackets are not needed.
- i prefer `pure` to `return`, but the wire code base disagrees.